actuator/rust/robstride/src/lib.rs

```diff
@@ -667,6 +667,33 @@ impl Motors {
             .get(&motor_id)
             .ok_or_else(|| std::io::Error::new(std::io::ErrorKind::NotFound, "No feedback found"))
     }
+
+    pub fn set_new_id(&mut self, motor_id: u8, new_motor_id: u8) -> Result<HashMap<u8, MotorFeedback>, std::io::Error> {
+        let motor_ids = self.motor_configs.keys().cloned().collect::<Vec<u8>>();
+
+        for id in motor_ids {
+            if id == motor_id {
+                let mut pack = CanPack {
+                    ex_id: ExId {
+                    id: motor_id,
+                    data: 0,
+                    mode: CanComMode::MotorId,
+                    res: 0,
+                },
+                len: 8,
+                data: vec![0; 8],
+                };
+                
+                pack.ex_id.data = ((new_motor_id as u16) << 8) | (motor_id as u16);
```

Comment:
this seems wrong - shouldn't it just be `new_motor_id` (not `motor_id`), also, i think we should be able to do something like `new_motor_id.to_le_bytes()`